### DISCUSSION TITLE
HBSCROLLの挙動について

### DISCUSSION THREAD

**Discussion by pansystar** [Tatelier/Tatelier#153] [General]
報告求む

**Comment by Reiya0917milk** [comment]


**Reply by pansystar** [reply]
これまたオモロイ譜面ですね！
見てみます。

**Reply by Reiya0917milk** [reply]
Mwfyuma Perustalloraの譜面にて
https://ux.getuploader.com/CurryDry0608hk_J/download/228
音源再生前に謎の譜面が描画されていて動作が重たくなる
音源再生後に動作が通常になる
1024行目の挙動が次郎と異なる
3585行目のカツ大が描画されない

**Reply by pansystar** [reply]
> Day by Day!の裏譜面で46行目のスクロールマイナスの部分が動いていませんでした。 https://ux.getuploader.com/oriiiiiiiiiiiiiiiiiii/download/1

この事象は対応方法の検討が難しいため、チケット化します。

**Reply by pansystar** [reply]
> 

本件、対応できました。
ただ、他の譜面にハレーションがないとは言えません...(涙)

yourbest nightmareの挙動は問題ありませんでしたが、もし他の譜面で挙動が変でしたら
改めて報告ください。

本日のリリースで対応
